.github/etc/R_Makevars_Linux

```diff
@@ -0,0 +1,5 @@
+CC=ccache gcc
+CXX=ccache g++
+CXX17=ccache g++ -Wno-deprecated-declarations
+#CXX17=ccache g++ 
+SHLIB_CXXLD=ccache g++
```

Comment:
It looks like this file is not used now?

Reply:
It used here:

https://github.com/eddelbuettel/mlpack/blob/c4b47ace3d6036c85fec4bea050a143bc8606084/.github/workflows/main.yml#L56

ie in a block

```yaml
  - name: Prepare R for Ccache 
        run: |
          mkdir -p ~/.R
          cp -vax .github/etc/R_Makevars_${{ runner.os }} ~/.R/Makevars
```

Reply:
Oops, I overlooked that!  For some reason I thought that was only using one of the Makevars files.